androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/nearbyTransit/NearbyTransitViewTest.kt

```diff
@@ -258,4 +262,50 @@ class NearbyTransitViewTest : KoinTest {
         composeTestRule.onNodeWithText("Green Line Head Sign").assertIsDisplayed()
         composeTestRule.onNodeWithText("5 min").assertIsDisplayed()
     }
+
+    @OptIn(ExperimentalTestApi::class)
+    @Test
+    fun testNearbyTransitViewNoNearbyStops() {
+        val emptyNearbyKoinApplication = koinApplication {
```

Comment:
thought(non-blocking): Making a ticket to explore an approach of [reducing the boilerplate of configuring Koin in tests](https://insert-koin.io/docs/reference/koin-android/instrumented-testing/)